resources/assets/app.scss

```diff
@@ -52,3 +52,8 @@ button.delete {
   font-weight: bold;
   margin-right: 5px;
 }
+
+.inbox-item {
+  padding: 48px 0;
```

Comment:
We could use [`$section-spacing`](https://github.com/DoSomething/forge/blob/3828463212a96e97b154b0afa007da30b869c5ab/scss/_utilities/_variables.scss#L99) for this.